ush/python_utils/environment.py

```diff
@@ -32,31 +32,30 @@ def str_to_date(s):
 
 
 def date_to_str(d, format="%Y%m%d%H%M"):
-    """Get string from python datetime object.
-    By default it converts to YYYYMMDDHHMM format unless
-    told otherwise by passing a different format
+    """Gets string from Python datetime object.
+    By default it converts to ``YYYYMMDDHHmm`` format unless told otherwise by passing a different format.
 
     Args:
-        d: datetime object
+        d (datetime.datetime): datetime object
```

Comment:
Should the d in datetime be capitalized?

```suggestion
        d (datetime.datetime): Datetime object
```